doc/src/manual/installation.md

```diff
@@ -18,7 +18,7 @@ On Windows Julia can be installed directly from the Windows store
 exactly the same version by executing
 
 ```
-winget install julia -s msstore
+winget install 9NJNWW8PVKMN -s msstore
```

Comment:
Does this need a `--id` to avoid triggering the same issue of causing a search? I.e. what if somebody creates a new app with `9NJNWW8PVKMN` in the description field. Also, can we give it both `julia` and the id, otherwise, it looks a little odd.

Reply:
I tried doing `winnget install julia --id 9NJNWW8PVKMN -s msstore` and it worked on my (US) machine

Reply:
What if you make the id wrong?

Reply:
Ugh still installs. There is “-e” flag for exact query but it’s not working

Reply:
Okay this works exactly: 
```
 winget install --name Julia --id 9NJNWW8PVKMN -e -s msstore
 ```

Reply:
Great. If you do a suggestion I'll accept it

Reply:
```suggestion
winget install --name Julia --id 9NJNWW8PVKMN -e -s msstore
```

Reply:
adding an exact search by name and id will ensure that we install the correct thing and doesnt look like gooblydegook